### ISSUE TITLE
Add support for printing current thread

### ISSUE THREAD

**Issue by VDKA** [SwiftyBeaver/SwiftyBeaver#51]
Printing the current thread would be useful information for debugging things (like core data). Similar to what is possible in XCGLogger. This is how they are doing it.

```swift
        if showThreadName {
            if NSThread.isMainThread() {
                extendedDetails += "[main] "
            }
            else {
                if let threadName = NSThread.currentThread().name where !threadName.isEmpty {
                    extendedDetails += "[" + threadName + "] "
                }
                else if let queueName = String(UTF8String: dispatch_queue_get_label(DISPATCH_CURRENT_QUEUE_LABEL)) where !queueName.isEmpty {
                    extendedDetails += "[" + queueName + "] "
                }
                else {
                    extendedDetails += "[" + String(format:"%p", NSThread.currentThread()) + "] "
                }
            }
        }
```

**Comment by skreutzberger** [comment]
Good idea, do you want to open a pull request? It should be very simple to do, just make sure that the call is done before SwiftyBeaver itself is entering its own thread.

**Comment by skreutzberger** [comment]
Thanks for the idea and the pull request! Closed with version 0.3.1 #81e81efa231f20ae6951e4d
